angular/main-es2015.js

```diff
@@ -0,0 +1 @@
Q.dd(3,"Basic checkbox"),Q.yc(),Q.zc(4,"section",1),Q.zc(5,"mat-checkbox"),Q.dd(6,"Check me!"),Q.yc(),Q.yc(),Q.yc(),Q.yc())},directives:[T.a,T.b,E.a],styles:[""]}),e})(),ee=(()=>{class e{constructor(){this.checked=!1,this.indeterminate=!1,this.labelPosition="after",this.disabled=!1}}return e.\u0275fac=function(t){return new(t||e)},e.\u0275cmp=Q.nc({type:e,selectors:[["checkbox-configurable-example"]],decls:27,vars:8,consts:[[1,"example-h2"],[1,"example-section"],[1,"example-margin",3,"ngModel","ngModelChange"],[1,"example-margin"],[3,"ngModel","ngModelChange"],["value","after",1,"example-margin"],["value","before",1,"example-margin"],[1,"result"],[1,"example-margin",3,"ngModel","indeterminate","labelPosition","disabled","ngModelChange","indeterminateChange"]],template:function(e,t){1&e&&(Q.zc(0,"mat-card"),Q.zc(1,"mat-card-content"),Q.zc(2,"h2",0),Q.dd(3,"Checkbox configuration"),Q.yc(),Q.zc(4,"section",1),Q.zc(5,"mat-checkbox",2),Q.Hc("ngModelChange",(function(e){return t.checked=e})),Q.dd(6,"Checked"),Q.yc(),Q.zc(7,"mat-checkbox",2),Q.Hc("ngModelChange",(function(e){return t.indeterminate=e})),Q.dd(8,"Indeterminate"),Q.yc(),Q.yc(),Q.zc(9,"section",1),Q.zc(10,"label",3),Q.dd(11,"Align:"),Q.yc(),Q.zc(12,"mat-radio-group",4),Q.Hc("ngModelChange",(function(e){return t.labelPosition=e})),Q.zc(13,"mat-radio-button",5),Q.dd(14,"After"),Q.yc(),Q.zc(15,"mat-radio-button",6),Q.dd(16,"Before"),Q.yc(),Q.yc(),Q.yc(),Q.zc(17,"section",1),Q.zc(18,"mat-checkbox",2),Q.Hc("ngModelChange",(function(e){return t.disabled=e})),Q.dd(19,"Disabled"),Q.yc(),Q.yc(),Q.yc(),Q.yc(),Q.zc(20,"mat-card",7),Q.zc(21,"mat-card-content"),Q.zc(22,"h2",0),Q.dd(23,"Result"),Q.yc(),Q.zc(24,"section",1),Q.zc(25,"mat-checkbox",8),Q.Hc("ngModelChange",(function(e){return t.checked=e}))("indeterminateChange",(function(e){return t.indeterminate=e})),Q.dd(26," I'm a checkbox "),Q.yc(),Q.yc(),Q.yc(),Q.yc()),2&e&&(Q.hc(5),Q.Qc("ngModel",t.checked),Q.hc(2),Q.Qc("ngModel",t.indeterminate),Q.hc(5),Q.Qc("ngModel",t.labelPosition),Q.hc(6),Q.Qc("ngModel",t.disabled),Q.hc(7),Q.Qc("ngModel",t.checked)("indeterminate",t.indeterminate)("labelPosition",t.labelPosition)("disabled",t.disabled))},directives:[T.a,T.b,E.a,r.g,r.i,L.b,L.a],styles:[".example-h2[_ngcontent-%COMP%]{margin:10px}.example-section[_ngcontent-%COMP%]{display:flex;align-content:center;align-items:center;height:60px}.example-margin[_ngcontent-%COMP%]{margin:0 10px}"]}),e})();n.d(t,"AppModule",(function(){return te}));let te=(()=>{class e{}return e.\u0275mod=Q.rc({type:e,bootstrap:[K,ee]}),e.\u0275inj=Q.qc({factory:function(t){return new(t||e)},providers:[{provide:Y.a,useValue:{appearance:"fill"}}],imports:[[s.a,c.b,r.c,o.b,J,a.l,r.j]]}),e})();Object(i.a)().bootstrapModule(te).catch(e=>console.error(e))},zn8P:function(e,t){function n(e){return Promise.resolve().then((function(){var t=new Error("Cannot find module '"+e+"'");throw t.code="MODULE_NOT_FOUND",t}))}n.keys=function(){return[]},n.resolve=n,e.exports=n,n.id="zn8P"}},[[0,0,5]]]);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Zone' is not defined.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5245121)